framework/doc/content/source/postprocessors/Residual.md

```diff
@@ -1,8 +1,16 @@
 # Residual
 
-!syntax description /Postprocessors/Residual
+This post-processor computes the nonlinear residual norm.
+The parameter [!param](/Postprocessors/Residual/residual_type) may have one of
+the following values:
+
+- `FINAL`: The final norm, obtained by `NonlinearSystemBase::finalNonlinearResidual()`
+- `INITIAL`: The post-SMO initial norm, obtained by `NonlinearSystemBase::preSMOResidual()`
+- `PRE_SMO`: The pre-SMO initial norm, obtained by `NonlinearSystemBase::initialResidual()`
```

Comment:
A user is likely going to have no idea what SMO means

Reply:
It's explained in that NonlinearSystem link. I'll move that link here